examples/platform/efr32/display/demo-ui-bitmaps.h

```diff
@@ -264,6 +264,36 @@
         0x00, 0xF8, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF,    \
         0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF,    \
         0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF
+#elif IS_DEMO_THERMOSTAT
+#define ON_DEMO_BITMAP                                                                                                             \
```

Comment:
why is this required to fix wifi builds?

The PR description says "fix wifi". Please update PR summary to explain the changes and what they do/why they are needed.

Reply:
Since we include the LCD files, might as well include the thermostat icon at the same time. Comment updated